### PR TITLE
Integrate sentry

### PR DESCRIPTION
**Pull request by apanjain**
Add sentry to track crashes in app

### CODE REVIEW

**Comment by hacksdump** [comment]
Can the sentry logs be shared within the team?

**Comment by apanjain** [comment]
> Can the sentry logs be shared within the team?
Yes, I'll add you

**Comment by hacksdump** [comment]
@apanjain conflict here.

**Comment by apanjain** [comment]
@hacksdump  Good to go

File: backend/backend/settings.py [hunk 1]
```diff
@@ -14,6 +14,19 @@
 
 import yaml
 
+# Sentry
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+
+sentry_sdk.init(
+    dsn="https://e5ef111a5b494bbaad2d780dc85e2223@o386884.ingest.sentry.io/5221627",
```
Comment (hacksdump) [inline backend/backend/settings.py:22]: This should rather be added to the conf.

Comment (apanjain) [reply backend/backend/settings.py]: done

**Review by hacksdump — APPROVED** [review]

**Pull request merged**